content/api.md

```diff
@@ -119,3 +254,7 @@ API version 2 is no longer available. Please upgrade to [version 3](#v3) if you
 ## Version 1 {#v1}
 
 API version 1 is no longer available. Please upgrade to [version 3](#v3) if you use this version.
```

Comment:
Are we keeping this and V2's message at "upgrade to version 3"?

Reply:
How about "upgrade to a supported version" with a link to the version table at the top?

Reply:
Sounds good to me.